plugins/modules/gitlab_hook.py

```diff
@@ -97,6 +97,11 @@
       - Trigger hook on wiki events.
     type: bool
     default: false
+  releases_events:
+    description:
+      - Trigger hook on release events.
+    type: bool
+    default: false
```

Comment:
The default must not be there. This basically breaks users who set this value manually without this module, and now the module suddenly changes the value for them.
```suggestion
    version_added: 8.4.0
```